### PR TITLE
Abdul/i2c port expander

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for I2C Port Expander app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : I2cPortExpander.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

**Review by rsameser — APPROVED** [review]

File: Resources/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (neilsh-msft) [inline Resources/.gitkeep]: Remove unused file.

**Review by neilsh-msft — COMMENTED** [review]
Resources/I2cPortExpanderDrawing_bb_750.png
 - picture wiring doesn't match readme file.  Picture has wires going to pins 1,3,5, and 6.  Readme has 27 and 28.  
 - expansion of 40 pin header is confusing, the red lines look like wires, needs cleanup.

File: I2CPortExpander/CS/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+namespace I2cPortExpander
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        // use these constants for controlling how the I2C bus is setup
+        private const string I2C_CONTROLLER_NAME = "I2C0"; //specific to Upboard
```
Comment (neilsh-msft) [inline I2CPortExpander/CS/MainPage.xaml.cs]: New string constant has been added but isn't used.  Please remove.

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: Resources/.gitkeep (hunk 1 above)
Comment (v-abdrahman) [reply Resources/.gitkeep]: Removed the file

File: I2CPortExpander/CS/MainPage.xaml.cs (hunk 2 above)
Comment (v-abdrahman) [reply I2CPortExpander/CS/MainPage.xaml.cs]: Removed the string

**Pull request merged**